entity-framework/core/what-is-new/ef-core-3.x/breaking-changes.md

```diff
@@ -475,6 +475,9 @@ This would still not be configured by convention to avoid misconfiguration when
 * **`DbContext.Query<>()`** - Instead `DbContext.Set<>()` should be used.
 * **`IQueryTypeConfiguration<TQuery>`** - Instead `IEntityTypeConfiguration<TEntity>`** should be used.
 
+> [!NOTE]
+> Due to [an issue in 3.x](https://github.com/dotnet/efcore/issues/19537) when querying keyless entities that have all properties set to `null` a `null` will be returned instead of an entity, if this is applicable to your scenario also add logic to handle `null` in results.
```

Comment:
```suggestion
> Due to [an issue in 3.x](https://github.com/dotnet/efcore/issues/19537) when querying keyless entities that have all properties set to `null` a `null` will be returned instead of an entity, if this issue is applicable to your scenario also add logic to handle `null` in results.
```